src/models/UDT/index.ts

```diff
@@ -17,3 +17,20 @@ export interface UDT {
   displayName?: string
   uan?: string
 }
+
+export enum MintStatus {
+  Minting = 'minting',
+  Closed = 'closed',
+  RebaseStart = 'rebase_start',
+}
+
+export interface OmigaInscription extends UDT {
+  mintStatus: MintStatus
+  mintLimit: string
+  expectedSupply: string
+  inscriptionId: string
+}
+
+export function isOmigaInscription(udt: UDT): udt is OmigaInscription {
+  return 'mintStatus' in udt
```

Comment:
It's prone to be covered by `sudt` or other token when they have `mint status` field for UI

Reply:
Expectedly, all UDT variants should have corresponding structures in the frontend, so there won't be incorrect judgments here.

However, a more comprehensive judgment would indeed be more robust. It has been updated.